app/Entry.js

```diff
@@ -18,7 +18,7 @@ import Onboarding1 from './views/onboarding/Onboarding1';
 import Onboarding2 from './views/onboarding/Onboarding2';
 import Onboarding3 from './views/onboarding/Onboarding3';
 import Onboarding4 from './views/onboarding/Onboarding4';
-import Onboarding5 from './views/onboarding/Onboarding5';
+import Onboarding5 from './views/onboarding/Onboarding4';
```

Comment:
import 5 from 4 ? intentional?

Reply:
If this is a intentional hack, write a comment about it.

Reply:
nope - good catch, my IDE didnt pick it up when i renamed the file.